website/docs/r/spring_cloud_gateway.html.markdown

```diff
@@ -67,6 +67,12 @@ resource "azurerm_spring_cloud_gateway" "example" {
     issuer_uri    = "https://www.test.com/issueToken"
     scope         = ["read"]
   }
+
+  response_cache {
+    cache_type   = "LocalCachePerRoute"
+    size         = "100MB"
+    time_to_live = "30s"
+  }
```

Comment:
this might be better/more accurately represent the API as
```suggestion
  local_response_cache_per_route {
    size         = "100MB"
    time_to_live = "30s"
  }
  
    local_response_cache_per_instance {
    size         = "100MB"
    time_to_live = "30s"
  }
```

instead of inventing a new type property & constant. we can use the same schema for these and ensure only 1 is allowed?